packages/container/tests/owner_test.js

```diff
@@ -0,0 +1,14 @@
+import { getOwner, setOwner } from 'container/owner';
+
+QUnit.module('Owner', {});
+
+QUnit.test('An owner can be set with `setOwner` and retrieved with `getOwner`', function() {
+  var owner = {};
+  var obj = {};
+
+  strictEqual(getOwner(obj), undefined, 'owner has not been set');
+
+  setOwner(obj, owner);
+
+  strictEqual(getOwner(obj), owner, 'owner has been set');
```

Comment:
should this test also assert the "symbol" nature of this `owner` property? To ensure it is non-enumerable, and also not collideable as `owner`